client/components/open/integrations/components/GoogleSheetsIntegrationActions.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <v-button
```

Comment:
Another thought no specified before: since there will be multiple users in workspaces very soon, it would be great here to be able to know what account is connected here, or at least which user created this connection. Maybe that's something we can put here ?
![CleanShot 2024-05-22 at 13 52 19@2x](https://github.com/JhumanJ/OpnForm/assets/11312432/d5d5d50f-9e40-4982-ab1c-3e676ffb7cb5)